### PR TITLE
Remove duplicate code and fix justify style issue

### PR DESCRIPTION
**Pull request by asturur**
A little bit of cleaning plus bugfix:

IText justified without style breakdown.

image

Fixes case in wich spaceWidth is infinite.

The justified spaces are not calculated right as of now, but I cannot fix this bug now.

### CODE REVIEW

**Comment by asturur** [comment]
@kangax ready to merge!

File: src/shapes/text.class.js [hunk 1]
```diff
@@ -471,29 +471,28 @@
...
-            widthDiff = totalWidth - wordsWidth,
-            numSpaces = words.length - 1,
-            spaceWidth = widthDiff / numSpaces,
-            leftOffset = 0;
-
-        for (var i = 0, len = words.length; i < len; i++) {
-          this._renderChars(method, ctx, words[i], left + leftOffset, top, lineIndex);
-          leftOffset += ctx.measureText(words[i]).width + spaceWidth;
+      // stretch the line
+      var words = line.split(/\s+/),
```
Comment (kangax) [inline src/shapes/text.class.js]: Hm, don't we have a separate method for this now?

Comment (asturur) [inline src/shapes/text.class.js]: if you mean splitting words, no i don t think so.
i can check.
anyway that code needs a review.

measuring width of words without calculating styles is very wrong and produce bad ( rare ) result.

Comment (asturur) [inline src/shapes/text.class.js]: no we do not have any method for splitting line in words.

**Pull request merged**